plugins/filter_aws/aws.c

```diff
@@ -434,7 +656,36 @@ static struct flb_config_map config_map[] = {
      " will be used: 'v1' or 'v2'. 'v2' may not work"
      " if you run Fluent Bit in a container."
     },
-
+    {
+     FLB_CONFIG_MAP_BOOL, "ec2_instance_type", "false",
```

Comment:
There should be config map properties for the existing fields, instance ID and AZ, so that a user can disable them

Reply:
Added them in